package.json

```diff
@@ -46,9 +46,6 @@
   "dependencies": {
     "apis-helpers": "0.0.1",
     "axios": "^0.9.1",
-    "babel": "^6.5.2",
-    "babel-core": "^6.7.4",
-    "babel-preset-es2015": "^6.5.0",
```

Comment:
Whatever I did, I couldn't get yarn to do what I wanted it to do, so I didn't commit the lock file. Can somebody else fix that for me?
(or ditch `yarn` completely and go with `npm@5` and `package-lock.json` files)

Reply:
I think `package-lock.json` files are the way to go but maybe in the next PR. Let me see if I can generate it.

Reply:
@MiniGod https://github.com/MiniGod/apis/pull/1

Reply:
You should be able to push to my fork, but thanks a lot for the PR. I've merged. :)